Description: Adaption of upstream DragonFlyBSD source
 GNU/kDragonFlyBSD resorts to a compatibility library.
Author: Mats Erik Andersson <debian@gisladisker.se>
Forwarded: not-needed
Last-Update: 2011-01-20

--- dragonflybsd-8.1.orig/usr.sbin/repquota/Makefile	2010-06-14 04:09:06.000000000 +0200
+++ dragonflybsd-8.1/usr.sbin/repquota/Makefile	2011-01-20 19:28:35.000000000 +0100
@@ -4,6 +4,8 @@
 PROG=	repquota
 MAN=	repquota.8
 
+LDADD=	-lbsd
+
 WARNS?=	4
 
 .include <bsd.prog.mk>
--- dragonflybsd-8.1.orig/usr.sbin/repquota/repquota.c	2010-06-14 04:09:06.000000000 +0200
+++ dragonflybsd-8.1/usr.sbin/repquota/repquota.c	2011-01-20 19:28:35.000000000 +0100
@@ -58,6 +58,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <bsd/string.h>
 #include <time.h>
 #include <unistd.h>
 #include <utmp.h>
